pcdsdevices/slits.py

```diff
@@ -643,3 +643,18 @@ class SimLusiSlits(LusiSlits):
     ywidth = Cpt(FastMotor)
     xcenter = Cpt(FastMotor)
     ycenter = Cpt(FastMotor)
+
+class JJSlits(SlitsBase):
+    '''
```

Comment:
Please add a descriptive string for the device:
* I believe JJ x-ray offers a variety of models of slits. Which does this specifically apply to?
* Is the class name as specific as it should be to disambiguate it from other slits on the beamline, and future JJ x-ray slits?

Reply:
@klauer this class is specifically for the AT-C8-HV model which have combined motors for a center motion, and a gap motion, as opposed to the other slits which have individual motors for each of the blades. Do you have any suggestions for a new class name?

Reply:
I think what you changed it to is reasonable. We're in the middle of trying to come up with a naming guide - like our style guide - so that there should be one unambiguous name to choose for a device.

Two things, though:
* This is still missing a docstring that includes the model number and relevant information
* The display filename will need to be renamed accordingly